app/src/main/java/com/sdp/movemeet/DistanceCalculator.java

```diff
@@ -16,7 +17,7 @@
     // This constant is more ~canonical~
     private final Double EARTH_RADIUS = 6.371 * Math.pow(10, 3);
     private Double userLatitude, userLongitude;
-    private ArrayList<Pair> activityDistanceMap = null;
+    private ArrayList<Pair> activityDistanceMap;
     private boolean sorted;
 
     public DistanceCalculator(Double userLatitude, Double userLongitude) {
```

Comment:
Maybe you can add a comment for explain how the class works

Reply:
Yep I agree. But since this is Mark's code, I'd rather let him do his own comments. I only deleted this "= null" while I was looking at his class, because it was not necessary